benchmark/run_trex.sh

```diff
@@ -24,14 +24,19 @@ args=(
 )
 get_args "$@"
 
+$verbose && echo "Running T-REX .."
+cmd="${trex_install_dir}/hgt3.4 -inputfile=${input_file_nwk}"
+$verbose && echo "Command:"$'\n'"  $cmd"
+
+printf "#TREX\n" >> $output_fp
```

Comment:
This is a good point. Not making any changes but `printf` sends a smell  of C/Perl complication. I replaced all `printf`s with `echo` or `echo -e`. @ekopylova If you have other considerations please let us know.